### PR TITLE
Created modules es6

### PR DESCRIPTION
**Pull request by BenjaminMatembe**
On this branch I:
- Added storage, bookcollectionClass, menuSelector files in the modules folder.

- Imported the above files in the entry file (index.js).

- Installed the luxon library and imported the DateTime in the entry file.

- Cleared linter errors

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1 +1,79 @@
...
+};
+
+setInterval(updateTime, 1000);
+
+// navbar
+
+for (let i = 0; i < menu.length; i += 1) {
+  menu[i].addEventListener('click', () => {
+    menuselector(menu[i]);
+  });
```
Comment (codepantha) [inline index.js:92]: - [ ] At the moment, due to the navigation links not working correctly, I am unable to test your Add and Remove books functionality. Whenever a navigation link is clicked an error occurs. Please see the console error below and try to fix it 🙏
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @BenjaminMatembe 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ No linter errors
✔️ Used ES6 modules 📌 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @BenjaminMatembe 👋, 

Great work on making the requested changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by BenjaminMatembe** [comment]
Hi @codepantha  Thank you for the given advise that i followed to implement the changes.
Thank you too for the approval.

**Pull request merged**